frontend/src/app/forms/steps/key-competencies/rating-modal/item-popover/item-popover.component.html

```diff
@@ -0,0 +1,37 @@
+<ion-header>
+  <ion-toolbar>
+    <ion-buttons slot="start">
+      <ion-button (click)="deleteItem()">
+        <ion-icon name="trash-outline" color="secondary"></ion-icon>
+      </ion-button>
+    </ion-buttons>
+
+    <ion-title color="primary">
+      {{ formItem.controls.name.value }}
+    </ion-title>
+
+    <ion-buttons slot="end">
+      <ion-button (click)="closePopover()">
+        Close
+      </ion-button>
+    </ion-buttons>
+  </ion-toolbar>
+</ion-header>
+
+<ion-content class="ion-padding">
+  <div class="description">
+    Please rate your skills from 1 (not good) to 5 (excellent).
+  </div>
+
```

Comment:
I don't think they should vote from "not good" to "excellent". I mean as an applicant, I never want to say, I'm not good in something. I have experience or i don't have one and then i will just not click on it and say, i have skills in it. So here would be instead of "not good" something like "basic" better, or maybe for the different key competencies different rating scales.

Reply:
Hm what about the following: We leave it as it is for now and then discuss in the team on how we should call it. Or even ask the Industry Partners. Would that be ok?